test/functional/vimscript/system_spec.lua

```diff
@@ -653,4 +653,34 @@ describe('shell :!', function()
         4]])
     end
   end)
+  it(':{range}! with powershell filter/redirect when command has arguments #19250', function()
```

Comment:
looks like we can just update the above test instead of adding an entirely new test. Just add `/R` to the above test.

Reply:
Done.